modules/rest-api/pages/rest-autocompact-per-bucket.adoc

```diff
@@ -0,0 +1,272 @@
+= Auto-Compaction: Per Bucket
+
+[abstract]
+Auto-compaction settings can be established and retrieved for individual buckets.
+
+== Description
+
+Auto-compaction settings can be modified for an existing bucket.
+Once this has occurred, the bucket's auto-compaction settings will no longer be affected by changes to the global auto-compaction settings.
+
+Auto-compaction settings for a specific bucket can be established and retrieved by the _Full Admin_ and _Cluster Admin_ roles.
+Auto-compaction settings for a specific bucket can be retrieved by a _Bucket Admin_ role that applies to that bucket; and by the _Read-Only Admin_ role.
+
+Auto-compaction settings for _GSI Indexes_ are provided; however, these are for use with Couchbase Server Community Edition only.
+These settings are not needed for the indexes used by Enterprise Edition: if these settings are established, they are ignored by nodes running Enterprise Edition.
+For more information, see xref:learn:services-and-indexes/indexes/storage-modes.adoc[Storage Settings].
+
+== HTTP methods and URIs
+
+Per bucket auto-compaction settings are retrieved and established by the following combinations of HTTP method and URI:
+
+----
+GET /pools/default/buckets/[bucket-name]
+
+POST /pools/default/buckets/[bucket-name]
+----
+
+== Curl Syntax
+
+----
+curl -X GET -u Administrator:password
+  http://[cluster-ip-address]:8091/pools/default/buckets/[bucket-name]
+
+curl -X POST -u Administrator:password
+  http://[cluster-ip-address]:8091/controller/setAutoCompaction
+    -d databaseFragmentationThreshold[percentage]=[percentage]
+    -d databaseFragmentationThreshold[size]=[size]
+    -d viewFragmentationThreshold[percentage]=[percentage]
+    -d viewFragmentationThreshold[size]=[percentage]
```

Comment:
in the `-d viewFragmentationThreshold[size]` it says [percentage] on the other side of the equals instead of size

Reply:
Thanks!

Reply:
Have modified below.